### PR TITLE
feat: JOIN-39891 added getPermissions function

### PR DESCRIPTION
**Pull request by denysovkos**
[JOIN-39891]

[JOIN-39891]: https://joinsolutionsag.atlassian.net/browse/JOIN-39891?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/roles-and-permissions/src/api/getPermissions.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const client = OpenFGAClient.getClient()
+  const response = await client.read({
+    object: formatCompany(attributes.companyId),
+    user: formatRoleWithPostfix(attributes.roleName, attributes.companyId, UserRelationsTypeEnum.ASSIGNEE),
+  })
+
+  return response.tuples.map(tup => tup.key.relation as AuthorizationRelationsType)
+}
+
+export default getPermissions
```
Comment (kirpichenko) [inline packages/roles-and-permissions/src/api/getPermissions.ts:22]: we don't usually use default exports

File: packages/roles-and-permissions/src/api/__tests__/getPermissions.test.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  it('returns permissions for admin role', async () => {
+    await initializeCompanyRolesAndPermissions({ companyId: 112, userId: 233 })
+    const result = await getPermissions({ companyId: 112, roleName: DefaultRolesEnum.ADMIN })
+    expect(result).toIncludeAllMembers(Object.values(AuthorizationRelationsTypeEnum))
+  })
+
+  it('returns permissions for custom role', async () => {
+    await initializeCompanyRolesAndPermissions({ companyId: 344, userId: 123 })
+    const openFgaClient = OpenFGAClient.getClient()
+    await openFgaClient.write({
```
Comment (kirpichenko) [inline packages/roles-and-permissions/src/api/__tests__/getPermissions.test.ts:29]: can we re-write a test without tuples deletion?  it's quite hard to understand.

Comment (denysovkos) [reply packages/roles-and-permissions/src/api/__tests__/getPermissions.test.ts:29]: at this moment we don't have any other roles, only admin. so to simulate this deletion some relations easiest way.
we can add this later, when function like `assignRole` will be implemented

File: packages/roles-and-permissions/src/api/getPermissions.ts (hunk 1 above)
Comment (denysovkos) [reply packages/roles-and-permissions/src/api/getPermissions.ts:22]: should I rewrite it to object export?

Comment (kirpichenko) [reply packages/roles-and-permissions/src/api/getPermissions.ts:22]: why object export? just export the method 
```
export const getPermissions: GetPermissionsAttributes = ...
```

**Review by kirpichenko — APPROVED** [review]

File: packages/roles-and-permissions/src/api/getPermissions.ts (hunk 1 above)
Comment (denysovkos) [reply packages/roles-and-permissions/src/api/getPermissions.ts]: fixed

**Pull request merged**